doc/references.bib

```diff
@@ -0,0 +1,34 @@
+@article{conformal_parameterization_2020,
+ author = {Choi, Gary P. T. and Leung-Liu, Yusan and Gu, Xianfeng and Lui, Lok Ming},
+ doi = {10.1137/19M125337X},
+ journal = {SIAM Journal on Imaging Sciences},
+ number = {3},
+ pages = {1049-1083},
+ title = {Parallelizable Global Conformal Parameterization of Simply-Connected Surfaces via Partial Welding},
+ volume = {13},
+ year = {2020}
+}
```

Comment:
Also here this reference should be removed. Also check below if we even use scipy in this project (I think not), then that reference can also be removed.